beast/tools/read_beast_data.py

```diff
@@ -132,39 +126,38 @@ def read_sed_data(
 
     Returns
     -------
-    grid_param_list : list of strings
-        if param_list is None, return the list of parameter options
+    Two possible returns depending on param_list input
 
-    beast_data: dictonary
-       contains arrays of the requested SED grid parameters
+    sed_data: dictonary (param_list input as list of strings)
+        contains arrays of the requested SED grid parameters
+        if return_params is True, then also provides
+
+    grid_param_list : list of strings (param_list input as None)
+        if param_list is None, return the list of parameter options
```

Comment:
I think I forgot to update the docstring when I changed this in #513. If you could update this to say when `return_params is True` rather that `param_list` is `None`, that would be awesome.

Reply:
Done.